tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -922,6 +922,7 @@ unsigned CGMSHLSLRuntime::ConstructStructAnnotation(DxilStructAnnotation *annota
     std::string fieldSemName = "";
 
     QualType fieldTy = fieldDecl->getType();
+    DXASSERT(!fieldDecl->isBitField(), "We should have already ensured we have no bitfields.");
```

Comment:
The reported bug looked like a memory corruption, presumably because the fields were assumed to be full-sized `uint`s even if marked with `: 1`. I added this so that if we remove the bitfields error, we'll hit this and should investigate the consequences in this code of allowing bitfields.